gui/src/2D/controls/container.ts

```diff
@@ -375,6 +375,7 @@ export class Container extends Control {
                         if (this.adaptHeightToChildren && child._height.isPixel) {
                             computedHeight = Math.max(computedHeight, child._currentMeasure.height + child.paddingTopInPixels + child.paddingBottomInPixels);
                         }
+                        this._markAsDirty();

```

Comment:
This should not happen every time. (only when computeWidth or height change)

Reply:
So setting inside markAsDirty() inside the ifs will not fix because the roto container which is the one that needs to be updated does not have adaptiveScaling. Does it not make sense if the children get updated then the root container will get updated as well?

Reply:
ok I get it so maybe calling it only when adaptivescaling is on. My point is that we need to figure out when to call markasdirty because we cannot call it per frame as it will defy to overall dirtiness feature ;D